### ISSUE TITLE
No learning_rates and xgb_model params in R version of xgb.train

### ISSUE THREAD

**Issue by flyingdisc** [dmlc/xgboost#819]
There are both params in Python version, but there are none in R version. 
-     **learning_rates** (list or function) – List of learning rate for each boosting round or a customized function that calculates eta in terms of current number of round and the total number of boosting round (e.g. yields learning rate decay) - list l: eta = l[boosting round] - function f: eta = f(boosting round, num_boost_round)
-     **xgb_model** (file name of stored xgb model or 'Booster' instance) – Xgb model to be loaded before training (allows training continuation). 

For the `xgb_model`, it's only a matter to add 2 lines of code to load model file, like this one in Python version by Far0n: https://github.com/Far0n/xgboost/commit/c768a042cba117547a10aa7682f7e6fd60cec6ea 

```
    if xgb_model is not None:
        bst.load_model(xgb_model)
```

Unfortunately I don't know how to add the code to the R package. Someone expert please add it to the R package ;) 

Thank you very much for this great software!
